localnet/kubernetes/values-relayminer-1.yaml

```diff
@@ -2,10 +2,8 @@ config:
   signing_key_name: supplier1
   suppliers:
     - service_id: anvil
-      type: http
       service_config:
-        url: http://anvil:8547/
-      proxy_names:
-        - http-proxy
-      hosts:
-        - tcp://relayminer1:8545
+        backend_url: http://anvil:8547/
```

Comment:
Thoughts on renaming this to `internal_service_url`?

cc @okdas

Reply:
@Olshansk tbh I like backend batter. In my mental model, it makes more sense.